rfc/transact_from_ethereum_to_substrate.md

```diff
@@ -0,0 +1,47 @@
+# RFC: Transact from Ethereum to Substrate
+
+
+## Summary
+
+This RFC proposes the feature to call transact from Ethereum to Substrate through our bridge, including two PRs separately with https://github.com/Snowfork/snowbridge/pull/1141 for solidity and https://github.com/Snowfork/polkadot-sdk/pull/114 for substrate. 
+
+## Explanation
+
+Basically it works as follows:
```

Comment:
I would add a fee flow section:

- dApp is represents `msg.sender` or its sovereign depending on context.
- Parachain represents the target parachain, its sovereign or its agent depending on context.

Sequence|Where|Who|What
-|-|-|-
1|Gateway|dApp|pays(ETH, converted to DOT here) Parachain Agent for delivery costs only (no execution cost)
2|Bridge Hub|Relayer|pays(DOT) node for execution
3|Bridge Hub|Parachain Sovereign|pays(DOT) Relayer for delivery (refund+reward)
4|Parachain|dApp|pays(DOT, Native) for execution only.

Does this balance? Yes because the parachain agent collects fees for delivery on Ethereum side and the Parachain Sovereign pays for delivery on the Bridge Hub side. 

Execution costs totally belongs to the dApp prefunding its sovereign on the Parachain.